csp/utils.py

```diff
@@ -95,3 +95,25 @@ def build_policy(config=None, update=None, replace=None, nonce=None):
 
     return '; '.join(['{} {}'.format(k, val).strip()
                       for k, val in policy_parts.items()])
+
+
+# Attempt to support all valid script tag attributes
```

Comment:
There is a bit of lingering weirdness here as `async` and `defer` are boolean but they support either a bare word or an `=true` to indicate a truthy value. Currently, a valid tag only accepts kwarg-style inputs so a bare word won't work and it seems like it's probably not worther the parser logic kartwheels to support mixed kwarg and arg style values. I'll at least make the discrepancy clear in the docs, but Im definitely open to trying to support it if anyone finds that too distasteful.